### ISSUE TITLE
Can't Install

### ISSUE THREAD

**Issue by tadashi-aikawa** [mrjackphil/obsidian-jump-to-link#20]
Hi, @mrjackphil !

I couldn't install obsidian-jump-to-link as follows.

![image](https://user-images.githubusercontent.com/9500018/121792430-88779680-cc2f-11eb-9957-f66951e00274.png)

Probably, it is because of the wrong [tag version 2.0.1](https://github.com/mrjackphil/obsidian-jump-to-link/releases/tag/2.0.1) despite [0.2.1 in manifest.json](https://github.com/mrjackphil/obsidian-jump-to-link/blob/master/manifest.json#L4).

Kind regards.

**Comment by mrjackphil** [comment]
Oh, thanks. Sorry for this. Ill fix it as soon as be able.

**Comment by mrjackphil** [comment]
Should be fixed in 0.2.2

**Comment by tadashi-aikawa** [comment]
@mrjackphil 
Thanks for your quick fix! I can install it now. 👍

And `activate-jump-to-anywhere` is so useful! I use this command with `:obcommand` on [Obsidian Vimrc Support Plugin](https://github.com/esm7/obsidian-vimrc-support#executing-obsidian-commands-with-obcommand). They are amazing! 🥳 

```
exmap jumpanywhere obcommand mrj-jump-to-link:activate-jump-to-anywhere
nmap s :jumpanywhere
```
